docs/articles/documentation/test-api/assertions/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: docs
+title: Assertions
+permalink: /documentation/test-api/assertions/
+checked: false
+---
+# Assertions
+
+To check if the state of the tested webpage matches the one you expect to see, use *assertions*.
+
+We will provide a comprehensive [set of assertions](assertion-api.md) and all of them based on Behavior Driven Development style (BDD-style).
```

Comment:
TestCafe provides a comprehensive [set of assertions](assertion-api.md) and all of them are based on Behavior Driven Development style (BDD-style).